config/flipt.schema.cue

```diff
@@ -210,6 +210,7 @@ import "strings"
 				password: string
 			}
 			poll_interval?: =~#duration | *"30s"
+			manifest_version?: "1.0" | "1.1"
```

Comment:
Quick suggestion to give it a default (not that anyone uses these I dont think):
```suggestion
			manifest_version?: "1.0" | *"1.1"
```